### PR TITLE
[feature] Copy user by holding

### PR DESCRIPTION
**Pull request by Florian-Sabonchi**
Added function which copies a username to the clipboard see #690

### CODE REVIEW

File: lib/views/widgets/modals/modal-user-details.dart [hunk 1]
```diff
@@ -179,10 +177,12 @@ class ModalUserDetails extends StatelessWidget {
                             overflow: TextOverflow.ellipsis,
                             textAlign: TextAlign.center,
                             style: Theme.of(context).textTheme.subtitle1,
-                          ), onTap: () async {
+                          ),
+                            onLongPress: () async  {
                               await Clipboard.setData(ClipboardData(text: props.user.userId));
-                              await props.onAddConfirmation('Username copied to clipboard');
-                          }),
+                              HapticFeedback.vibrate();
```
Comment (EdGeraghty) [inline lib/views/widgets/modals/modal-user-details.dart:183]: Nice use of Haptic feedback. I'll probably add that to the other copyable fields!

Comment (EdGeraghty) [reply lib/views/widgets/modals/modal-user-details.dart:183]: I do think we should still pop up a confirmation, though. Let's see what others think.

**Review by ereio — APPROVED** [review]

**Comment by ereio** [comment]
good work @Florian-Sabonchi!

**Comment by Florian-Sabonchi** [comment]
> good work @Florian-Sabonchi!

Thanks for merging

**Pull request merged**